packages/testcontainers/src/test-container.ts

```diff
@@ -63,6 +65,7 @@ export interface StopOptions {
 export interface StartedTestContainer {
   stop(options?: Partial<StopOptions>): Promise<StoppedTestContainer>;
   restart(options?: Partial<RestartOptions>): Promise<void>;
+  commit(options: ContainerCommitOptions): Promise<string>;
```

Comment:
Could we swap these type names around so that `CommitOptions` is used here in the public API, and `ContainerCommitOptions` is used internally? I suggest it mostly for consistency, as all options are implicitly container options